### PR TITLE
libjxl: add package

### PR DESCRIPTION
**Pull request by star-hengxing**

### CODE REVIEW

File: packages/l/libjxl/xmake.lua [hunk 1]
```diff
@@ -5,6 +5,8 @@ package("libjxl")
     set_license("BSD-3-Clause")
 
     set_urls("https://github.com/libjxl/libjxl.git")
+    add_urls("https://gitlab.com/wg1/jpeg-xl.git")
```
Comment (waruqi) [inline packages/l/libjxl/xmake.lua:8]: 直接传到一个 add_urls 就行了

Comment (waruqi) [reply packages/l/libjxl/xmake.lua:8]: 我是说用一个 add_urls

```
add_urls("https://github.com/libjxl/libjxl.git",
         "https://gitlab.com/wg1/jpeg-xl.git")
```

**Comment by waruqi** [comment]
> CMAKE_CUSTOM_LINKER -m64 -fno-rtti -funwind-tables -Xclang -mrelax-all -Xclang -mconstructor-aliases -fno-omit-frame-pointer -O3 -DNDEBUG -O2 -m64 -fPIE -pie CMakeFiles/hwy_list_targets.dir/hwy/tests/list_targets.cc.o -o hwy_list_targets  libhwy.a 
Error running link command: No such file or directory

@xq114 没被替换掉么？https://github.com/xmake-io/xmake-repo/pull/2134

**Comment by xq114** [comment]
> CMake Warning:
>   Manually-specified variables were not used by the project:
> 
>     CMAKE_ASM_COMPILER
>     CMAKE_ASM_FLAGS
>     CMAKE_CUSTOM_LINKER

看起来 CMAKE_CUSTOM_LINKER 没成功传进去，干脆不要这个变量了，CMAKE_CXX_LINK_EXECUTABLE 里面直接写linker完整命令，`envs.CMAKE_CXX_LINK_EXECUTABLE = "/usr/bin/clang <FLAGS> <CMAKE_CXX_LINK_FLAGS> <LINK_FLAGS> <OBJECTS> -o <TARGET> <LINK_LIBRARIES>"`

**Comment by waruqi** [comment]
现在写死可以了

**Pull request merged**